web_frontend/osmaxx/excerptexport/tests/views/test_views_with_dowload_links.py

```diff
@@ -0,0 +1,122 @@
+from unittest.mock import patch
+from uuid import UUID
+
+import pytest
+from django.contrib.auth.models import Group
+from django.core.urlresolvers import reverse
+from django.test.testcases import SimpleTestCase
+
+from osmaxx.excerptexport.models.bounding_geometry import BBoxBoundingGeometry
+from osmaxx.contrib.auth.frontend_permissions import FRONTEND_USER_GROUP
+from osmaxx.excerptexport.forms.order_options_mixin import available_format_choices
+from osmaxx.excerptexport.models.excerpt import Excerpt
+from osmaxx.excerptexport.models.extraction_order import ExtractionOrder, ExtractionOrderState
+from osmaxx.excerptexport.models.output_file import OutputFile
+
+
+@pytest.fixture
+def authenticated_client(client, django_user_model):
+    user = django_user_model.objects.create_user(username='lauren', password='lauri', email=None)
+    client.login(username='lauren', password='lauri')
+    client.user = user
+    return client
+
+
+@pytest.fixture
+def authorized_client(authenticated_client):
+    authenticated_client.user.groups.add(Group.objects.get(name=FRONTEND_USER_GROUP))
+    return authenticated_client
```

Comment:
I'd like to move above fixtures to a shared location (`conftest.py`?), but I think I'll need some help with that.